docs/howto/catalog-queries.md

```diff
@@ -12,6 +12,9 @@ Then you can query the catalog by using `curl` commands and the `jq` CLI tool to
     By default, Catalogd is installed with TLS enabled for the catalog webserver.
     The following examples will show this default behavior, but for simplicity's sake will ignore TLS verification in the curl commands using the `-k` flag.
 
+!!! note 
```

Comment:
this doesn't render correctly, I think the syntax is [!NOTE], or maybe [!IMPORTANT]?

Similar case for https://github.com/operator-framework/operator-controller/pull/1841/files#diff-6295b5bd58ccf02e628fe42adbc43f32b9df3f3ad4ab61ac813366162d65a490R54 and https://github.com/operator-framework/operator-controller/pull/1841/files#diff-944a5a8340da19a891934706c3a6a9f5eeded7eacdce3c107edd1f9dfdd7f173R36

Reply:
resolved in https://github.com/operator-framework/operator-controller/pull/1841#discussion_r1995038039